src/js/components/modals/AppModalComponent.jsx

```diff
@@ -68,6 +69,7 @@ var AppModalComponent = React.createClass({
     AppFormStore.on(FormEvents.CHANGE, this.onFormChange);
     AppFormStore.on(FormEvents.FIELD_VALIDATION_ERROR,
       this.onFieldValidationError);
+    Mousetrap.bind(["command+enter", "ctrl+enter"], () => this.handleSubmit());
```

Comment:
That's neat. We should probably add a message at the bottom of the form at some point, documenting this. Something like `Press ⌘ + Enter to submit`